### DISCUSSION TITLE
build SU2 in zsh

### DISCUSSION THREAD

**Discussion by meokyo** [su2code/SU2#1641] [Q&A]
I am a beginner of coding. When I tried to build SU2 from source code, the commond:

./meason.py build 

the result is open it via vim mode.
I think the reason is my shell is zsh.
Finally I run bash and it works.
I would like to know which is the code in zsh...

**Comment by martin-g** [comment]
Is `python3` in the $PATH in zsh ?

`meson.py` starts with this shebang line `#!/usr/bin/env python3`. So, if you start it with `./meson.py` then the shell will read the first line and pass it to the python3 interpreter. If python3 is not in the path then maybe you have some fallback that opens it in $EDITOR ...

**Reply by meokyo** [reply]
Hi martin-g, 
in bash, the reply of _which python3_ is **/usr/local/bin/python3**, and the reply of _whereis python3_ is **python3: /usr/bin/python3** and _.bash_profile_ doesn't have $PATH of that.
In Zsh, the reply of _whereis python3_ is **python3: /usr/bin/python3 /Library/Frameworks/Python.framework/Versions/3.10/share/man/man1/python3.1** and the reply of _which python3_ is **/Library/Frameworks/Python.framework/Versions/3.10/bin/python3**, and .zshrc deesn't have $PATH of that.
What should I do with the condition?

**Reply by martin-g** [reply]
Maybe it is some MacOS file association...
No idea.
